assets/src/edit-story/app/uploader/test/useUploader.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks';
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import ConfigContext from '../../config/context';
+import useUploader from '../useUploader';
+
+function setup(args, refreshLibrary = false) {
+  const configValue = {
+    api: {},
+    allowedMimeTypes: {
+      image: ['image/png', 'image/jpeg', 'image/jpg', 'image/gif'],
+      video: ['video/mp4'],
+    },
+    allowedFileTypes: ['png', 'jpeg', 'jpg', 'gif', 'mp4'],
+    maxUpload: 104857600,
+    capabilities: {
+      hasUploadMediaAction: true,
+    },
+    ...args,
+  };
+  const wrapper = (params) => (
+    <ConfigContext.Provider value={configValue}>
+      {params.children}
+    </ConfigContext.Provider>
+  );
+  const { result } = renderHook(() => useUploader(refreshLibrary), { wrapper });
+  return {
+    uploadFile: result.current.uploadFile,
+    isValidType: result.current.isValidType,
+  };
+}
+
+describe('useUploader', () => {
+  it('user unable to upload', () => {
```

Comment:
Nit, but I usually write tests so that it results in a sentence like `_it_ throws an error when user does not have upload permissions`, as that's how the default test reporter displays tests.

Reply:
(That also applies to other tests in this PR, not just this particular one)